package.json

```diff
@@ -83,6 +89,7 @@
     "hardhat-preprocessor": "0.1.4",
     "husky": "6.0.0",
     "inquirer": "8.1.0",
+    "lint-staged": ">=10",
```

Comment:
precise versioning 🙏

Reply:
I've now tagged this dependency to version **12.5.0** which is the maximum allowed version for node engine **12.x**